.github/workflows/build.yaml

```diff
@@ -0,0 +1,69 @@
+---
+name: Test container builds
+
+"on":
+  pull_request:
+
+jobs:
+  kics-scan:
+    name: KICS Scan
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          path: "repo"
+      - name: KICS Github Action
+        uses: checkmarx/kics-action@v1.5
+        with:
+          path: "repo"
+          output_path: "results"
+          exclude_queries: ${{ env.KICS_user_is_root }}
+          fail_on: high,medium
```

Comment:
looks like there was a medium result, but it didn't fail

Reply:
Yeah this workflow doesn't yet have enough permissions to write or fail as it's a new workflow added by an external contributor.